src/scripts/events.js

```diff
@@ -0,0 +1,15 @@
+import lyricsOnDom from '../components/lyricsOnDom';
+
+const events = () => {
+  const form = document.querySelector('form');
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const artistEntry = document.querySelector('#artistInput').value;
+    const songEntry = document.querySelector('#songInput').value;
+    lyricsOnDom(artistEntry, songEntry);
+    form.reset();
+    document.querySelector('#artistTitle').innerHTML = artistEntry;
```

Comment:
Remember, we want to keep our events and dom manipulation separate. SO though this is simple, you would want to either ruse a render to dom function or pull this out into its separate function like lyric on the dom.